docs/source/command_line.md

```diff
@@ -0,0 +1,209 @@
+
+| xbrlDB                                                              | Saves XBRL data to a database.                                                                                                                                                                                                                                                                                          |
+|---------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| \--store-to-XBRL-DB                                                 | Store into XBRL DB.  Provides connection string: host,[port], user, password, database, [timeout], technology.                                                                                                                                                                                                          |
+|                                                                     | postgres – XBRL-US Postgres SQL<br/> mssqlSemantic – Semantic MSSQL SQL<br/> mysqlSemantic – Semantic MySQL SQL<br/>  orclSemantic – Semantic Oracle SQL<br/> pgSemantic – Semantic Postgres SQL<br/>rexster – Rexter (Titan Cassandra)<br/>rdfDB – RDF (Turtle, NanoSparqlServer)<br/>json – JSON (JSON, MongoDB)<br/> | 
+|                                                                     | For rdfDB if hostname is rdfTurtleFile or rdfXmlFile, then database specifies a local file system pathname to store a file.  <br/>For json, if hostname is jsonFile then database specifies a local file system path name to store into a file.<br/>                                                                    |
+| Example storing into postgres database using XBRL-US schema         | arelleCmdLine -f c:\temp\test.rss -v –disclosureSystem efm-pragmatic-all-years –store-to-XBRL-DB “myhost.com,8084,pgUserId,pgPassword,test_db,90,postgres”                                                                                                                                                              |
+| Example storing into local MSSQL database using SQLEXPRESS instance | arelleCmdLine -f c:\temp\test.rss -v –disclosureSystem efm-pragmatic-all-years –store-to-XBRL-DB “localhost\SQLEXPRESS,,sqlLogin,sqlPassword,,90,mssqlSemantic”                                                                                                                                                         |
+
+### Disclosure System Selections
+
+The disclosure systems are documented in the config directory file disclosuresystems.xml.  These are current entries.
```

Comment:
Should probably mention that this is an incomplete list defined by plugins and that they aren't available unless you enable their respective plugins (also gfm is known to be broken). It might be better to just inform folks to run `--disclosureSystem help` to list the disclosure systems that are available for the plugins they have enabled?